resources/definitions/TL-D4Pro.def.json

```
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "name": "TL-D4Pro",
+    "inherits": "tenlog_printer",
+    "metadata": {
+	"visible": true,
+        "manufacturer": "Tenlog",
+        "platform": "TL-D4Pro.3mf",
+        "platform_offset": [-700,285,-252]
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "TL-D4P" },
+        "machine_width": {
+            "default_value": 400
+        },
+        "machine_depth": {
+            "default_value": 400
+        },
+        "machine_height": {
+            "default_value": 400
+        },
+        "prime_tower_position_x": {
+            "value": "200"
+        },
+	    "prime_tower_position_y": {
```

Mix of tabs and spaces
```suggestion
        "prime_tower_position_y": {
```